### PR TITLE
Clarified Ambiguity regarding Advancement Conditions #5661

### PR DESCRIPTION
**Pull request by notrichardpeng**
I'm not sure if making the `attempt_result` strings not independently usable would cause any problems. 
Also, I'm not sure how this change would be updated for translations other than English, should I try to do that part as well?

### CODE REVIEW

File: WcaOnRails/lib/advancement_condition.rb [hunk 1]
```diff
@@ -83,12 +83,13 @@ def self.wcif_type
   end
 
   def to_s(round, short: false)
+    round_format = I18n.t("formats#{".short" if short}.#{round.format_id}")
     if round.event.timed_event?
-      I18n.t("advancement_condition#{".short" if short}.attempt_result.time", time: SolveTime.centiseconds_to_clock_format(attempt_result))
+      round_format + I18n.t("advancement_condition#{".short" if short}.attempt_result.time", time: SolveTime.centiseconds_to_clock_format(attempt_result))
```
Comment (Jambrose777) [inline WcaOnRails/lib/advancement_condition.rb:88]: For our locals, you can have multiple inputs in inject into the string. The strings in en.yml should be fully readable on their own. So for example, The en.ymal variable can look like:
`time: "%{round_format} < %{time}"`
and then the response in the to_s method can look like:
`I18n.t("advancement_condition#{".short" if short}.attempt_result.time", round_format: round_format, time: SolveTime.centiseconds_to_clock_format(attempt_result))`

**Review by Jambrose777 — COMMENTED** [review]
> Also, I'm not sure how this change would be updated for translations other than English, should I try to do that part as well?

For translations, we only have to update the en.yml file. Doing this will send a notification to our translators indicating that there is new material to translate. So nothing more would need to be done to achieve that from our end.

File: WcaOnRails/lib/advancement_condition.rb (hunk 1 above)
Comment (notrichardpeng) [reply WcaOnRails/lib/advancement_condition.rb:88]: Thanks for pointing this out! I didn't know it could be used that way.

**Comment by notrichardpeng** [comment]
Is this cool now? Should I merge it?

**Review by Jambrose777 — APPROVED** [review]
LGTM!

**Pull request merged**